python/pyspark/ml/image.py

```diff
@@ -48,7 +48,7 @@ def imageSchema(self):
         """
         Returns the image schema.
 
-        :rtype StructType: a DataFrame with a single column of images
+        :return: StructType: a DataFrame with a single column of images
```

Comment:
Let's remove the type specification here -> `` :return: a :class:`StructType` ... blabla ``

Reply:
BTW, output doc can be checked:

```
pip install sphinx
cd python/docs
make clean
make html
# open up "file:///.../spark/python/docs/_build/html/pyspark.ml.html#module-pyspark.ml.image" in your browser.
```

Reply:
done, and thank you for the useful tip on the documentation!